apps/teams-test-app/src/components/PagesConfigAPIs.tsx

```diff
@@ -106,7 +106,28 @@ const RegisterOnRemoveHandler = (): React.ReactElement =>
     },
   });
 
-const RegisterOChangeConfigHandler = (): React.ReactElement =>
```

Comment:
Similarly, I love that you are fixing this typo, but you have to keep the old named element around until you update the cypress tests to look for the correctly spelled element.

Reply:
Correct me if I'm wrong, but I think fixing this typo should be fine since the cypress test actually looks for `config_registerChangeConfigsHandler` by examining the DOM

Reply:
Oh right, good catch!

Reply:
this is a merge mistake, my git lens was showing that I added an 'O' so I thought maybe I was the one that introduced the typo.  My mistake